Topics/Tech_Stacks.md

```diff
@@ -1 +1,28 @@
 ## Tech Stacks
+1. React Native
+    1. Set up
+        1. Installation
+            1. Install Node.js and npm (Node Package Manager) on your computer. You can download and install Node.js from the official website: https://nodejs.org/en/
+                #Note: Suggested to download the LTS version because it is generally more stable than the Current or Latest version.
+            2. Install expo go on your ios device (https://apps.apple.com/us/app/expo-go/id982107779) or android device (https://play.google.com/store/apps/details?id=host.exp.exponent&hl=en_CA&gl=US&pli=1)
+            3. Download VS Code Extensions
+                1. ES7+ React/Redux/React-Native snippets (Extension ID: dsznajder.es7-react-js-snippets)
+                2. React Native Tools (Extension ID: msjsdiag.vscode-react-native)
+        2. Command
+            1. Install expo
+                npm install -g expo-cli
+            2. Initialize expo
+                expo init your_app_name
+                #Note: you will be able to choose "blank", "blank (TypesScript)", "tabs (TypeScript)", or "minimal". I personally prefer blank because often times I need to delete files I don't want. 
+            3. Start expo
+                cd your_app_name
+                expo start
+        3. Running expo
```

Comment:
Is it possible for you to format the inline code similar to how [MySQL page](https://learning-software-engineering.github.io/Topics/Tech_Stacks/Learning_MySQL.html) was done?

It's hard to tell what code we should run on first look:
![image](https://user-images.githubusercontent.com/37385317/226799956-6c28dcef-12c1-4d07-b8b7-4a3469bf8b5c.png)